packages/vite/src/node/server/transformRequest.ts

```diff
@@ -31,7 +31,7 @@ const debugCache = createDebugger('vite:cache')
 
 export interface TransformResult {
   code: string
-  map: SourceMap | null
+  map: SourceMap | { mappings: '' } | null
```

Comment:
This change is a breaking change.
But this type was not correctly declared and actually `transformRequest` could return `{ mappings: '' }` if a `load` hook returned a map with that and no transform hook transformed that request.